src/EA.Weee.Web/Areas/Scheme/Views/TransferEvidence/_ViewTransferEvidenceNotePartial.cshtml

```diff
@@ -71,11 +72,20 @@
         <p aria-labelledby="recipient" class="govuk-body weee-word-break">@Html.Raw(Model.RecipientAddress)</p>
     </div>
 
-    @if (Model.EditMode)
+    @if (Model.EditMode && Model.HasBeenReturned)
```

Comment:
Should this be an Or, as user should be able to edit the transfer without it being returned... looks like it wont go in unless both are true